torch/csrc/jit/serialization/export.cpp

```diff
@@ -957,6 +964,11 @@ std::tuple<std::string, RawDataExportMap> export_onnx(
       add_node_names,
       use_external_data_format,
       onnx_file_path);
+  const size_t proto_size = graph_encoder.get_model_proto().ByteSizeLong();
```

Comment:
I am wondering whether model_proto will be larger than 2GB, or just directly truncated when dumping to string.

Reply:
nvm, not a valid question